### ISSUE TITLE
aria-description and the rule "ARIA attribute is defined in WAI-ARIA" (5f99a7)

### ISSUE THREAD

**Issue by dan-tripp-siteimprove** [auto-wcag/auto-wcag#2366]
Periodically I see one of our customers use the [aria-description attribute from the ARIA 1.3 draft](https://www.w3.org/TR/wai-aria-1.3/#aria-description) then complain when we fail it under rule [5f99a7](https://www.w3.org/WAI/standards-guidelines/act/rules/5f99a7/).  The latest such customer used as a precedent: axe gives it a pass.  I still pushed back.  The ACT rule mentions the ARIA version that it tests for: 1.2.  Apparently that's not enough.  Maybe we should sort out, even more explicitly, how this rule should handle an aria attribute that is defined in an ARIA draft.  Perhaps with a failed example or two.  Suggestions welcome.

For what it's worth, based on my tests today, QualWeb and IBM fail it.  [Here's a test page](https://dan-tripp-siteimprove.github.io/aria-abcdef-test-page--2025-10-22--IwcURe8.html).

**Comment by carlosapaduarte** [comment]
CG decision: Update the definition of WAI-ARIA specification to reflect that new versions of ARIA may introduce or remove attributes/roles/states and the implementor might chose to support those or not
